services/openshift/scripts/openshift

```diff
@@ -73,6 +73,12 @@ wait_for_config_files() {
 master_config=${OPENSHIFT_DIR}/master-config.yaml
 node_config=${ORIGIN_DIR}/openshift.local.config/node-$(hostname)/node-config.yaml
 
+# Block which respect openshift_option
```

Comment:
what i am saying/asking is whether there is no a better way to structure the script? Otherwise a more descriptive comment would be nice